clients/company-supervisor/src/routes/Company.tsx

```diff
@@ -1,12 +1,78 @@
-import React from 'react';
-import { Header } from '../components/Header';
+import React, { useEffect, useState } from 'react';
```

Comment:
React 17 sürümünden sonra React'ı import etmeye gerek kalmadı istinasi durumlar dışında. [reference](https://www.chakshunyu.com/blog/why-did-and-dont-you-need-to-import-react/)